embedded/EscapeROOM/radio_morse_panel/radio_morse_panel.c

```diff
@@ -22,148 +44,79 @@ void unit()
 	}
 }
 
-void dot()
-{
+void playDot(){
 	carrier = CARRIER_ON;
-	unit();
+	playUnit();
 	carrier = CARRIER_OFF;
 }
 
-void dash()
-{
+void playDash(){
 	carrier = CARRIER_ON;
-	unit(); unit(); unit();
+	playUnit(); playUnit(); playUnit();
 	carrier = CARRIER_OFF;
 }
 
-void short_space()
-{
-	unit();
-}
-
-void letter_space()
-{
-	unit(); unit(); unit();
+void playShortSpace(){
+	playUnit();
 }
 
-void word_space()
-{
-	unit(); unit(); unit(); unit(); unit(); unit(); unit();
-}
-
-void zero()
-{
-	dash(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash();
-}
-
-
-void one()
-{
-	dot();  short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash();
+void playLetterSpace(){
+	playUnit(); playUnit(); playUnit();
 }
 
-
-void two()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dash(); short_space();
-	dash(); short_space();
-	dash();
-}
-
-
-void three()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dash(); short_space();
-	dash();
-}
-
-
-void four()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dash();
-}
-
-
-void five()
-{
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot();
+void playWordSpace(){
+	playUnit(); playUnit(); playUnit(); playUnit(); playUnit(); playUnit(); playUnit();
 }
 
-void six()
-{
-	dash(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot();
-}
-
-void seven()
-{
-	dash(); short_space();
-	dash(); short_space();
-	dot(); short_space();
-	dot(); short_space();
-	dot();
+char* findMorseChar(char letter, const MorseCode* morseAlphabet){
+	for(unsigned char i = 0; morseAlphabet[i].symbol != 0; i++){
+		if( morseAlphabet[i].symbol == letter)
+			return morseAlphabet[i].code;
```

Comment:
You are mixing brace styles here. After "for" statement you have single statement "if" inside curled braces, but after "if" statement you omit curled braces. It is nothing critical - just a small inconsistency. AND please note - sometimes small inconsistency in coding style might lead to more-easy-to-read code.
However I recommend to be consistent about your chosen style.
Either 
```
	for (unsigned char i = 0; morseAlphabet[i].symbol != 0; i++) {
		if (morseAlphabet[i].symbol == letter) {
			return morseAlphabet[i].code;
                }
        }
```
or 
```
	for (unsigned char i = 0; morseAlphabet[i].symbol != 0; i++)
		if (morseAlphabet[i].symbol == letter)
			return morseAlphabet[i].code;
```


Next comes my commentary/toughs/experience about different curled brace styles.
All programmers could be divided into multiple camps.
Camp A: always use curled braces {} after flow control statements (if. while, do, for);
```
if (expression) {
    statement();
}
// or
if (expression)
{
    statement();
}
```
Camp B: omit curled braces whenever possible, but put conditional statement in next line;
```
if (expression)
    statement();
```
Camp C: write whole statement in one line if possible; 
```
if (expression) statement();
```


Problem might occur when you need to add next statement2() under same if /for/while:
Camp A: curled braces already in place, just add another statement;
```
if (expression) {
    statement();
    statement2();
}
// or
if (expression)
{
    statement();
    statement2();
}
```
Camp B: must add curled braces
```
if (expression) {
    statement();
    statement2();
}
// OR disaster scenario:
if (expression)
    statement();
    statement2(); // <----- hard-to spot error introduced. When you read quickly you might not notice that
                  //        there is no curled braces and statement2() is always executed. Sometimes I spot
                  //        such error only when I step thru program with debugger and have that "AHHH"
                  //        moment when I see that statement(2) is executed even when expression is false.
```
Camp C: must add curled braces: 
```
if (expression) { statement(); statement2(); }
// OR disaster scenario:
if (expression) statement(); statement2(); // This too can be hard-to-spot error - especially with long expressions and statements;   
```